gulpfile.ts

```diff
@@ -58,7 +58,7 @@ function startsWith(value: string, prefix: string): boolean {
 }
 
 function endsWith(value: string, suffix: string): boolean {
-  return value && suffix && value.lastIndexOf(suffix) === value.length - suffix.length;
```

Comment:
we have [string.endsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith) available at runtime of this script right?

Reply:
I got errors that it wasn't defined because for some reason this is defaulting to use es5. I wasn't sure how to specify that this file should use es6, so I just wrote the method for endsWith().

Reply:
But yes, string.endsWith() should be available at runtime.